doc/en/customize.rst

```diff
@@ -14,15 +14,102 @@ configurations files by using the general help option:
 This will display command line and configuration file settings
 which were registered by installed plugins.
 
+.. _`config file formats`:
+
+Configuration file formats
+--------------------------
+
+Many :ref:`pytest settings <ini options ref>` can be set in a *configuration file*, which
+by convention resides on the root of your repository or in your
+tests folder.
+
+A quick example of the configuration files supported by pytest:
+
+pytest.ini
+~~~~~~~~~~
+
+``pytest.ini`` files take precedence over other files, even when empty.
+
+.. code-block:: ini
+
+    # pytest.ini
+    [pytest]
+    minversion = 6.0
+    addopts = -ra -q
+    testpaths =
+        tests
+        integration
+
+
+pyproject.toml
+~~~~~~~~~~~~~~
+
+.. versionadded:: 6.0
+
+``pyproject.toml`` are considered for configuration when they contain a ``tool.pytest.ini_options`` table.
+
+.. code-block:: toml
+
+    # pyproject.toml
+    [tool.pytest.ini_options]
```

Comment:
is there a reason for `.ini_options` (curious) -- most others I've seen are just `tool.X`

Reply:
It's a extra key since it's just a new way to spell the legacy options, we haven't yet made actual native toml support